cyberdrop_dl/managers/config_manager.py

```diff
@@ -160,17 +160,15 @@ def _verify_settings_config(self) -> None:
         default_settings_data = copy.deepcopy(settings)
         existing_settings_data = _load_yaml(self.settings)
         self.settings_data = _match_config_dicts(default_settings_data, existing_settings_data)
-        paths = {
-            (
-                [
-                    ("Files", "input_file"),
-                    ("Files", "download_folder"),
-                    ("Logs", "log_folder"),
-                    ("Sorting", "sort_folder"),
-                    ("Sorting", "scan_folder"),
-                ]
-            )
-        }
+        paths = set(
+            [
+                ("Files", "input_file"),
+                ("Files", "download_folder"),
+                ("Logs", "log_folder"),
+                ("Sorting", "sort_folder"),
+                ("Sorting", "scan_folder"),
+            ]
+        )
         enums = {("Dupe_Cleanup_Options", "hashing"): Hashing}
         for key, value in default_settings_data.items():
             for subkey, subvalue in value.items():
```

Comment:
This code block will be deleted after merging the pydantic branch. We can ignore the ruff error